06-statistical-testing.Rmd

```diff
@@ -225,15 +225,15 @@ If we want an 80% confidence interval for the test statistic, we can use the fun
 
 ```{r}
 #| label: stattest-ttest-ex1-ci80
-confint(ttest_ex1, level = 0.95)
-confint(ttest_ex1, level = 0.8)
+confint(ttest_ex1, level = 0.95) 
```

Comment:
The output from this line and then next are different.  This one has headers, but the next one does not.  Is there a reason for this?  Can we fix this or at least identify the reason for the reader?

Reply:
I've added an explanation - please review this.